### ISSUE TITLE
Gettings requirements to build wheel  did not run successfully

### ISSUE THREAD

**Issue by cbout22** [google/bumble#115]
Hello,
I'm facing an issue while installing the newest Bumble version ( > 0.0.134) on a fedora Linux 37 VM.

The following commands give me an error (see below):
```bash
pip install bumble
python -m pip install bumble
```
even while cloning and trying a local install with `pip` and `python` local cmd lines such as:
```bash
pip install -e .
```

error :
```bash
Collecting libusb-package==1.0.26.0
  Downloading libusb-package-1.0.26.0.tar.gz (408 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 408.7/408.7 kB 3.1 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [18 lines of output]
.
.
.
        File "/tmp/pip-build-env-8ff85nug/overlay/lib/python3.11/site-packages/setuptools/build_meta.py", line 335, in run_setup
          exec(code, locals())
        File "<string>", line 20, in <module>
      ModuleNotFoundError: No module named 'tomli'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

I found one way to fix it: a newest version of libusb-package (1.0.26.1 works on my VMs).

In `setup.cfg` : **`libusb-package == 1.0.26.0; platform_system!='Emscripten'` $\implies$ `libusb-package == 1.0.26.1; platform_system!='Emscripten'`**

**Comment by barbibulle** [comment]
Thanks for reporting this. I have pushed a hot fix for it, and created a package release v0.0.140. Please let me know if that works for you.

**Comment by cbout22** [comment]
Thanks for the hotfix, it works!
